Dockerfile.arm

```diff
@@ -312,6 +312,7 @@ COPY --from=rust-env /usr/local/rustup /usr/local/rustup
 
 
 # RUN rm -rf /usr/local/cargo /usr/local/rustup
+RUN rm -rf /root/.cache/bazel
```

Comment:
If the idea was to reduce image size, you should put this command in the same `RUN` where the cache is generated. Docker layers are additive; so that `.cache` directory would still be shipped; alongside the layer removing it 😄

Reply:
This was actually to fix CVEs from the bazel cache